README.md

```diff
@@ -6,7 +6,91 @@ For more details read the [TSC Charter](https://github.com/nodejs/TSC/blob/maste
+
+The TSC is responsible for a number of projects that are not strictly required to plan, test, build, document and ship Node.js releases. Projects that are _adjacent_ are either created from within the technical organization managed by the TSC or are adopted into that organization from outside.
+
+In the case of adopting existing projects, once the TSC has decided that adoption appropriate, it should seek agreement from the Node.js Foundation Board for such adoption as it impacts on the scope of technical activities of the Foundation.
+
+Current adjacent projects that are within the TSC's scope of responsibility can be found at the following locations:
+
+* https://github.com/nodejs/api — administration delegated to the API Working Group
+* https://github.com/nodejs/benchmarking — administration delegated to the Benchmarking Working Group
+* https://github.com/nodejs/code-and-learn
+* https://github.com/nodejs/diagnostics — administration delegated to the Diagnostics Working Group
+* https://github.com/nodejs/docs — administration delegated to the Documentation Working Group
+* https://github.com/nodejs/education
+* https://github.com/nodejs/email — administration delegated to the Build Working Group
+* https://github.com/nodejs/evangelism — administration delegated to the Evangelism Working Group
+* https://github.com/nodejs/github-bot — administration delegated to the Core Technical Committee
+* https://github.com/nodejs/help — administration delegated to the Core Technical Committee
+* https://github.com/nodejs/http — administration delegated to the HTTP Working Group
+* https://github.com/nodejs/http2 — administration delegated to the Core Technical Committee
+* https://github.com/nodejs/iojs.org — administration delegated to the Website Working Group
+* https://github.com/nodejs/installer — administration delegated to the Core Technical Committee
+* https://github.com/nodejs/Intl — administration delegated to the Intl Working Group
+* https://github.com/nodejs/llnode — administration delegated to the Core Technical Committee
+* https://github.com/nodejs/nan — administration delegated to the Addon API Working Group
+* https://github.com/nodejs/node-addon-examples — administration delegated to the Addon API WorkingGroup)
+* https://github.com/nodejs/node-chakracore — administration delegated to the Core Technical Committee
+* https://github.com/nodejs/node-gyp — administration delegated to the Core Technical Committee
+* https://github.com/nodejs/nodereport — administration delegated to the Post Mortem Working Group
+* https://github.com/nodejs/post-mortem — administration delegated to the Post Mortem Working Group
+* https://github.com/nodejs/promises — administration delegated to the Promises Working Group
+* https://github.com/nodejs/readable-stream — administration delegated to the Streams Working Group
+* https://github.com/nodejs/summit
+* https://github.com/nodejs/testing — administration delegated to the Testing Working Group
```

Comment:
probably need to add

https://github.com/nodejs/abi-stable-node - administration delegated to the API Working Group
https://github.com/nodejs/abi-stable-node-addon-examples - administration delegated to the API Working Group